src/Servers/Connections.Abstractions/src/IConnectionListener.cs

```diff
@@ -8,12 +8,28 @@
 
 namespace Microsoft.AspNetCore.Connections
 {
+    /// <summary>
+    /// Defines an interface that represents a listener bound to a specific <see cref="EndPoint"/>.
+    /// </summary>
     public interface IConnectionListener : IAsyncDisposable
     {
+        /// <summary>
+        /// The end point that was bound. This may differ from the requested end point.
```

Comment:
Inconsistent use of `end point` vs `endpoint`. Looks like [endpoint is a word](https://www.lexico.com/en/definition/endpoint) and that's what we've used in other places, so let's just be consistent.

```suggestion
        /// The endpoint that was bound. This may differ from the requested endpoint, such as when the caller requested that any free port be selected.
```

Good to include an example of why it would be different too.

Reply:
If you'd rather not provide the example (I'm not super worried either way):

```suggestion
        /// The endpoint that was bound. This may differ from the requested endpoint.
```